hvantk/tests/_snapshot_utils.py

```
@@ -0,0 +1,334 @@
+        # (recurses into tlocus above for locus<rg>-typed intervals).
+        start = _jsonable_to_hail_python(value["start"], dtype.point_type)
+        end = _jsonable_to_hail_python(value["end"], dtype.point_type)
+        return hl.Interval(start=start, end=end, includes_start=True, includes_end=False)
+    if isinstance(dtype, hl.tarray):
+        return [_jsonable_to_hail_python(v, dtype.element_type) for v in value]
+    if isinstance(dtype, hl.tset):
+        return {_jsonable_to_hail_python(v, dtype.element_type) for v in value}
+    if isinstance(dtype, hl.ttuple):
+        return tuple(
+            _jsonable_to_hail_python(v, t)
+            for v, t in zip(value, dtype.types)
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)